src/t_zset.c

```diff
@@ -2621,7 +2621,7 @@ void zunionInterDiffGenericCommand(client *c, robj *dstkey, int numkeysIndex, in
             {
                 j++; remaining--;
                 withscores = 1;
-            } else if (cardinality_only && remaining >= 2 &&
+            } else if (remaining >= 2 &&
```

Comment:
please update the json files of all relevant commands (that now take LIMIT) and `make` in order to update commands.c